data_structures/hash_table/hash_table.py

```diff
@@ -0,0 +1,48 @@
+"""Hash Table that works off of my premade linked list."""
+
+from ..linked_list.linked_list import LinkedList
+
+
+class HashTable:
+    """Hash Table."""
+
+    def __init__(self, max_size=1024):
+        """Hash Table initilization."""
+        self.max_size = max_size
+        self.buckets = [LinkedList() for _ in range(max_size)]
+
+    def _hash_key(self, val):
+        """Get the hashed key."""
+        return sum(map(lambda x: ord(x), val)) % self.max_size
+
+    def set(self, key, val):
+        """Insert into hash table."""
+        return self.buckets[self._hash_key(key)].insert({key: val})
+
+    def get(self, key):
+        """Get value."""
+        current = self.buckets[self._hash_key(key)].head
+        while current:
+            if key in current.val.keys():
+                return current.val[key]
```

Comment:
If the same key has been added to the HashTable twice, this will only find the first one.